src/LogDensityProblems.jl

```diff
@@ -13,6 +13,7 @@ documentation.
 module LogDensityProblems
 
 export ADgradient
+Base.@deprecate_moved ADgradient "LogDensityProblemsAD"
```

Comment:
I wonder if this is useful or if we could just tag a breaking release?

It seems `@deprecate_moved` makes `ADgradient` throw an error - but since it is specialized for the different backends, actually users will not see the error (which would be breaking BTW) unless they call `ADgradient` incorrectly. (As a minor point, `@deprecate_moved` by default already exports `ADgradient`: https://github.com/JuliaLang/julia/blob/d2aedf48ca799cbe11c6db9188e8af9ee5340e5f/base/deprecated.jl#L196-L205).

Reply:
Good point. I will close this, and prepare a PR for a breaking release.